search/Iterative_deepening_search_8_tile_puzzle.cpp

```diff
@@ -0,0 +1,197 @@
+#include <bits/stdc++.h>
+using namespace std;
```

Comment:
```suggestion
```
Please do not use bits/stdc++ as it is not portable code
And namespace std is bad practice

Please read the contribution guidelines to see how to contribute effectively

Reply:
did changes in recent PR